examples/Folders/Readme.md

```diff
@@ -0,0 +1,49 @@
+Dropzone that accepts folders as drag-and-drop.
+
+```
+const {getDroppedOrSelectedFiles} = require('html5-file-selector')
```

Comment:
It looks like this code didn't run through prettier before committing.  AFAIK it should have white spaces inside `{}`